### DISCUSSION TITLE
23.11.07 데일리 스크럼

### DISCUSSION THREAD

**Discussion by minju412** [petit-a-petit/MIML#56] [Daily Scrum]
.

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - 친구 수락 기능
    - PR
- 내일은 어떤 일을 하나요?
    - 리뷰 확인 및 코드 수정
    - 포스트맨 에러 해결 (다음 PR)
    - 테스트 코드 작성 (다음 PR)

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 아티스트와 유저 엔티티 관계 결정(N:N)
  - 알림 기능 클래스 다이어그램 작성 #19 
  - Spotify에서 어떤 기준으로 정보를 가져올지?
- 내일은 어떤 일을 하나요?
  - Spotify 제공 데이터 더 알아보기

**Comment by Nine-JH** [comment]
* 오늘은 어떤 일을 했나요?
* 내일은 어떤 일을 하나요?
  * 테스트 코드 작성
  * 3D Secure 살펴보기
